### PR TITLE
Router manufacturers

### PR DESCRIPTION
**Pull request by secretrobotron**
This pull request includes documentation published by various router manufacturers, including those worthy of review (at CR) in the future.

### CODE REVIEW

File: rules/ampedwireless.com.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<sitename name="ampedwireless.com">
+  <docname name="Terms and Conditions">
+    <url name="https://www.ampedwireless.com/legal#terms" xpath="//div[@id='terms']" lang="EN">
+      <norecurse name="arbitrary"/>
+    </url>
+  </docname>
+  <docname name="Privacy Policy">
+    <url name="https://www.ampedwireless.com/legal#privacys" xpath="//div[@id='privacy']" lang="EN">
```
Comment (secretrobotron) [inline rules/ampedwireless.com.xml]: Crawler isn't able to find this one. The [linked page](https://www.ampedwireless.com/legal#privacy) hides the specified element until you click on something in the side bar, but it should be in the DOM regardless. Any tips?

File: rules/eero.com.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<sitename name="eero.com">
+  <docname name="Terms of Service">
+    <url name="https://eero.com/legal/tos" xpath="//div[@class='legal-content']" lang="EN">
+      <norecurse name="arbitrary"/>
+    </url>
+  </docname>
+  <docname name="Privacy policy for eero Devices, Applications and Services">
+    <url name="https://eero.com/legal/privacy" xpath="//div[@class='legal-content']" lang="EN">
```
Comment (secretrobotron) [inline rules/eero.com.xml]: These should be `//div[contains(@class, 'legal-content')]`

File: rules/ampedwireless.com.xml (hunk 1 above)
Comment (secretrobotron) [reply rules/ampedwireless.com.xml]: Ah, `//*[@id='privacy']` works.

Comment (michielbdejong) [reply rules/ampedwireless.com.xml]: great!

File: rules/eero.com.xml (hunk 2 above)
Comment (michielbdejong) [reply rules/eero.com.xml]: So  do you want to change that or should we merge it as it is now, with `//div[@class='legal-content']`?

Comment (michielbdejong) [reply rules/eero.com.xml]: Ah right, you did that in c72b6c33d5a - great!

**Comment by michielbdejong** [comment]
I ssh'ed into the tosback.org server and did a `git pull` there, so the new rules should be crawled on the next daily run, in about 10 hours form now. Thanks a lot for your contributions!  <3

**Comment by secretrobotron** [comment]
Merci beaucoup! More to come. 🥇 

cc @lingjief

**Pull request merged**